basket/settings.py

```diff
@@ -341,11 +342,49 @@ def get_default_gateway_linux():
 K8S_NAMESPACE = config("K8S_NAMESPACE", default=None)
 K8S_POD_NAME = config("K8S_POD_NAME", default=None)
 
+# Data scrubbing before Sentry
+# https://github.com/laiyongtao/sentry-processor
+SENSITIVE_FIELDS_TO_MASK_ENTIRELY = [
+    "custom_id",
+    "email",
+    "first_name",
+    "ip_address",
+    "last_name",
+    "mobile_number",
+    "payee_id",
+    "remote_addr",
+    "remoteaddresschain",
+    "token",
+    "user",
+    "x-forwarded-for",
+]
+
+SENSITIVE_FIELDS_TO_MASK_PARTIALLY = [
+    "amo_id",
+    "fxa_id",
+    "id",
+    "uid",
+]
+
+
+def before_send(event, hint):
+    processor = DesensitizationProcessor(
+        with_default_keys=True,
+        sensitive_keys=SENSITIVE_FIELDS_TO_MASK_ENTIRELY,
+        partial_keys=SENSITIVE_FIELDS_TO_MASK_PARTIALLY,
+        mask_position=POSITION.LEFT,
+        off_set=3,
+    )
+    event = processor.process(event, hint)
```

Comment:
style nit: any reason to not just `return processor.process(event, hint)` here?

Reply:
Just a habit where the variable name helps hint at what's being returned, that's all.